action.yml

```diff
@@ -19,6 +19,10 @@ inputs:
     description: 'The number of suggestions per comment (smaller numbers work better for heavy pull requests)'
     required: false
     default: '10'
+  repo_path_prefix:
+    description: 'The path to repo when codes are analyzed with clang-tidy'
```

Comment:
Can you briefly specify the exact use case here?
We need to illustrate that users typically don't need that.

Reply:
> We need to illustrate that users typically don't need that.

For users don't need that, the default setting works just like the former hard-coded bash script, and the implement details keep opaque for users.

> Can you briefly specify the exact use case here? 

It's necessary for users who run `clang-tidy` in a docker container. I considered this before, but it might be too long to put the introduction in the "description" field. Maybe providing a showcase for it in `readme.md`? What do you think?

Reply:
You can add something like `may be needed for users who run clang-tidy in a docker container`.
I understand that the default value will work for most, however those who dive in deeper shouldn't lose time looking into a parameter that may not concern them. :)

Also, can you also change `when codes are analyzed` to `when code is analyzed`? 🙏

Reply:
@platisd Sure! I've updated instructions as you suggested. Sorry for my English.